playbooks/roles/os_temps/tasks/add_rh_cert.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Add RH IT Cert
+# This is required to pull the OpenShift 3.11 Jenkins image from
+# docker-registry.upshift.redhat.com.  Remove this task after
+# upgrading to Jenkins OCP4 (CVP-2930)
+
+# get kubeadmin password
+- name: "Get kubeadmin password"
+  set_fact:
+    kubeadmin_password: "{{ lookup('file', '~/.crc/machines/crc/kubeadmin-password') }}"
+
+# login as admin to the cluster
+- name: "Login as admin to the Openshift cluster"
+  command: "{{ oc_bin }} login https://api.crc.testing:6443 --username=kubeadmin --password={{ kubeadmin_password }} --insecure-skip-tls-verify=true"
+
+# Add Red Hat CA certs to OpenShift
+- name: "Add Red Hat CA certs to OpenShift"
+  command: "placeholder"
```

Comment:
@14rcole wouldnt this fail. ?
by placeholder command not found. 
I think you should add a debug task here instead.

Reply:
It will, I'm going to replace it with a task that reads the provided certs file, a task that generates the `oc patch` json, and a task that runs the command (which would be this task).  This won't be merged yet

Reply:
The command has been updated